style.css

```diff
@@ -374,3 +401,273 @@ form {
+      url("images/background_images/Frame2.png") 5% 30% no-repeat,
+      url("images/background_images/Frame3.png") 75% 0% no-repeat,
+      url("images/background_images/Frame4.png") 40% 80% no-repeat,
+      url("images/background_images/Frame5.png") 80% 100% no-repeat,
+      url("images/background_images/Frame6.png") 15% 95% no-repeat,
+      url("images/background_images/Group\ 35.png") 0 0 no-repeat;
+    background-size: 180px, 160px, 150px, 75px, 150px, 100px, cover;
+    width: 100%;
+  }
+
+  .bar {
+    display: none;
+  }
+
+  .greeting {
+    display: none;
+  }
+
+  .greeting-desktop {
+    display: block;
+    margin-top: 62.3px;
+    margin-bottom: 0;
+    color: #172b4d;
+  }
+
+  .desktop-view-heading {
+    display: block;
+  }
+
+  .mob-view-heading {
+    display: none;
+  }
+
+  .about_section {
+    background:
+      url("images/background_images/illustration-about-me-2-desktop.png") 0% 10% no-repeat,
+      url("images/background_images/illustration-about\ me-1-desktop.png") 100% 0% no-repeat;
+    background-size: 150px, 150px;
+  }
+
+  .about_section p {
+    width: 60%;
+    font-size: 17px;
+    height: auto;
+    line-height: 25px;
+  }
+
+  .contact_section {
+    background:
+      url("images/background_images/illustration-1-contact-form-desktop.png") 0% 95% no-repeat,
+      url("images/background_images/Frame-contact.png") 100% 95% no-repeat,
+      url("images/background_images/illustration-2-contact-form-desktop.png") 20% 90% no-repeat;
+    background-size: 140px, 160px, 100px;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .email {
+    display: none;
+  }
+
+  .contact_section h2 {
+    font-size: 22px;
+    text-align: inherit;
+    line-height: 32px;
+    width: 90%;
+    font-style: normal;
+    font-weight: 700;
+    margin: 120px 5px 0 80px;
+  }
+
+  form {
+    width: 100%;
+    margin: 80px 80px 190px 0;
+  }
+
+  input,
+  textarea {
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    padding: 8px;
+    margin: 6px;
+    width: 90%;
+  }
+
+  .contact_section button {
+    padding: 10px;
+    font-size: 14px;
+    margin: 0;
+  }
+
+  .works_section {
+    display: grid;
+    grid-template-rows: 80px 1fr 1fr;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 45px 66px 66px 55px;
+    box-sizing: border-box;
+  }
+
+  .bottom-separator {
+    display: none;
+  }
+
+  .details h2 {
+    margin: 10px;
+    line-height: 25px;
+    font-size: 18px;
+  }
+
+  .work-item {
+    height: 300px;
+  }
+
+  .works-heading {
+    grid-column: 1/4;
+  }
+
+  .tech-stack li {
+    font-weight: 600;
+    font-size: 8px;
+    padding: 1px 5px;
+    color: #3a4a42;
+  }
+
+  .tech-stack {
+    justify-content: space-around;
+    margin: 0 5px;
+    box-sizing: border-box;
+  }
+
+  .project button {
+    padding: 8px;
+    font-size: 12px;
+    color: #fff;
+    border: none;
+  }
```

Comment:
- [ ] [OPTIONAL] Considering your mastery and taste for good design, I think you might like to implement button states of interactivity to improve the quality of your design and user experience. States like the `:hover` and `:active` states. Attached is a recommended color palette with suitable colors for all states:

![image](https://user-images.githubusercontent.com/99715640/236335543-8fa455b2-e296-4f16-b2a8-97bf0aa433aa.png)